
Mon, 06/14/21, 07:15:01 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 07:15:01 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 07:15:01 PM:  Complete! 

Mon, 06/14/21, 07:15:01 PM: 
    Validating user authentification...

Mon, 06/14/21, 07:15:01 PM:  Complete! 

Mon, 06/14/21, 07:15:01 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 07:15:10 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 135, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 338, in exe_menu
    op_choice = input( PROG_INTRO )
KeyboardInterrupt

Mon, 06/14/21, 07:15:10 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 07:16:51 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 07:16:51 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 07:16:51 PM:  Complete! 

Mon, 06/14/21, 07:16:51 PM: 
    Validating user authentification...

Mon, 06/14/21, 07:16:52 PM:  Complete! 

Mon, 06/14/21, 07:16:52 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 07:16:53 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 135, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 338, in exe_menu
    op_choice = input( PROG_INTRO )
KeyboardInterrupt

Mon, 06/14/21, 07:16:53 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 07:17:07 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 07:17:07 PM: 
    Validating user authentification...

Mon, 06/14/21, 07:17:07 PM:  Complete! 

Mon, 06/14/21, 07:17:07 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : false          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 07:17:13 PM: 
Attempting program start...

Mon, 06/14/21, 07:17:13 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 07:17:13 PM:  Complete! 

Mon, 06/14/21, 07:17:13 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 07:17:14 PM:  Complete! 

Mon, 06/14/21, 07:17:14 PM: 
    Getting issue data...

Mon, 06/14/21, 07:17:40 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 135, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 352, in exe_menu
    issue_metalist = get_issue_info( session, issue_paged_list,
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 761, in get_issue_info
    if comments_paged_list.totalCount == 0:
  File "/home/j/.local/lib/python3.9/site-packages/github/PaginatedList.py", line 149, in totalCount
    headers, data = self.__requester.requestJsonAndCheck(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 354, in requestJsonAndCheck
    *self.requestJson(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 454, in requestJson
    return self.__requestEncode(cnx, verb, url, parameters, headers, input, encode)
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 528, in __requestEncode
    status, responseHeaders, output = self.__requestRaw(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 555, in __requestRaw
    response = cnx.getresponse()
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 127, in getresponse
    r = verb(
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1345, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 268, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.9/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.9/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
KeyboardInterrupt

Mon, 06/14/21, 07:17:40 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:05:29 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:05:29 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:05:29 PM:  Complete! 

Mon, 06/14/21, 09:05:29 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : false          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:07:58 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 135, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 339, in exe_menu
    op_choice = input( PROG_INTRO )
KeyboardInterrupt

Mon, 06/14/21, 09:07:58 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:07:59 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:07:59 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:07:59 PM:  Complete! 

Mon, 06/14/21, 09:07:59 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : false          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:10:11 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 135, in main
    try:
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 339, in exe_menu
    # get operation choice
KeyboardInterrupt

Mon, 06/14/21, 09:10:11 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:10:27 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:10:27 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:10:28 PM:  Complete! 

Mon, 06/14/21, 09:10:28 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : false          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:10:32 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 136, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 340, in exe_menu
    op_choice = input( PROG_INTRO )
KeyboardInterrupt

Mon, 06/14/21, 09:10:32 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:12:39 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:12:39 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:12:39 PM:  Complete! 

Mon, 06/14/21, 09:12:39 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : false          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:12:40 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 136, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 340, in exe_menu
    op_choice = input( PROG_INTRO )
KeyboardInterrupt

Mon, 06/14/21, 09:12:40 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:12:51 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:12:51 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:12:51 PM:  Complete! 

Mon, 06/14/21, 09:12:51 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:12:52 PM:  Complete! 

Mon, 06/14/21, 09:12:52 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:13:01 PM: 
Attempting program start...

Mon, 06/14/21, 09:13:01 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 09:13:01 PM:  Complete! 

Mon, 06/14/21, 09:13:01 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 09:13:02 PM:  Complete! 

Mon, 06/14/21, 09:13:02 PM: 
    Getting issue data...

Mon, 06/14/21, 09:13:06 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 136, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 354, in exe_menu
    issue_metalist = get_issue_info( session, issue_paged_list,
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 767, in get_issue_info
    for comment in comments_paged_list:
  File "/home/j/.local/lib/python3.9/site-packages/github/PaginatedList.py", line 56, in __iter__
    newElements = self._grow()
  File "/home/j/.local/lib/python3.9/site-packages/github/PaginatedList.py", line 67, in _grow
    newElements = self._fetchNextPage()
  File "/home/j/.local/lib/python3.9/site-packages/github/PaginatedList.py", line 199, in _fetchNextPage
    headers, data = self.__requester.requestJsonAndCheck(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 354, in requestJsonAndCheck
    *self.requestJson(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 454, in requestJson
    return self.__requestEncode(cnx, verb, url, parameters, headers, input, encode)
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 528, in __requestEncode
    status, responseHeaders, output = self.__requestRaw(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 555, in __requestRaw
    response = cnx.getresponse()
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 127, in getresponse
    r = verb(
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1345, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 268, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.9/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.9/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
KeyboardInterrupt

Mon, 06/14/21, 09:13:06 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:13:35 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:13:35 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:13:35 PM:  Complete! 

Mon, 06/14/21, 09:13:35 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:13:36 PM:  Complete! 

Mon, 06/14/21, 09:13:36 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:13:48 PM: 
Attempting program start...

Mon, 06/14/21, 09:13:48 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 09:13:48 PM:  Complete! 

Mon, 06/14/21, 09:13:48 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 09:13:48 PM:  Complete! 

Mon, 06/14/21, 09:13:48 PM: 
    Getting issue data...

Mon, 06/14/21, 09:13:54 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 136, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 354, in exe_menu
    issue_metalist = get_issue_info( session, issue_paged_list,
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 746, in get_issue_info
    issue_name_str  = cur_issue_user.name
  File "/home/j/.local/lib/python3.9/site-packages/github/NamedUser.py", line 276, in name
    self._completeIfNotSet(self._name)
  File "/home/j/.local/lib/python3.9/site-packages/github/GithubObject.py", line 297, in _completeIfNotSet
    self._completeIfNeeded()
  File "/home/j/.local/lib/python3.9/site-packages/github/GithubObject.py", line 301, in _completeIfNeeded
    self.__complete()
  File "/home/j/.local/lib/python3.9/site-packages/github/GithubObject.py", line 308, in __complete
    headers, data = self._requester.requestJsonAndCheck("GET", self._url.value)
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 354, in requestJsonAndCheck
    *self.requestJson(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 454, in requestJson
    return self.__requestEncode(cnx, verb, url, parameters, headers, input, encode)
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 528, in __requestEncode
    status, responseHeaders, output = self.__requestRaw(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 555, in __requestRaw
    response = cnx.getresponse()
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 127, in getresponse
    r = verb(
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1345, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 268, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.9/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.9/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
KeyboardInterrupt

Mon, 06/14/21, 09:13:54 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:14:12 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:14:12 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:14:12 PM:  Complete! 

Mon, 06/14/21, 09:14:12 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:14:12 PM:  Complete! 

Mon, 06/14/21, 09:14:12 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:14:16 PM: 
Attempting program start...

Mon, 06/14/21, 09:14:16 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 09:14:16 PM:  Complete! 

Mon, 06/14/21, 09:14:16 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 09:14:17 PM:  Complete! 

Mon, 06/14/21, 09:14:17 PM: 
    Getting issue data...

Mon, 06/14/21, 09:14:54 PM:  Complete! 

Mon, 06/14/21, 09:14:54 PM: 
    Writing list of issue data to JSON...

Mon, 06/14/21, 09:14:54 PM: 
    Specified directory does not exist! Creating directory... 

Mon, 06/14/21, 09:14:54 PM:  Complete! 

Mon, 06/14/21, 09:14:54 PM:  Complete! 

Mon, 06/14/21, 09:14:54 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:19:29 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:19:29 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:19:29 PM:  Complete! 

Mon, 06/14/21, 09:19:29 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:19:30 PM:  Complete! 

Mon, 06/14/21, 09:19:30 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 100
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:19:37 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 136, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 340, in exe_menu
    op_choice = input( PROG_INTRO )
KeyboardInterrupt

Mon, 06/14/21, 09:19:37 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:19:52 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:19:52 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:19:52 PM:  Complete! 

Mon, 06/14/21, 09:19:52 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:19:52 PM:  Complete! 

Mon, 06/14/21, 09:19:52 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 10
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:19:53 PM: 
Attempting program start...

Mon, 06/14/21, 09:19:53 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 09:19:53 PM:  Complete! 

Mon, 06/14/21, 09:19:53 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 09:19:53 PM:  Complete! 

Mon, 06/14/21, 09:19:53 PM: 
    Getting issue data...

Mon, 06/14/21, 09:19:57 PM:  Complete! 

Mon, 06/14/21, 09:19:57 PM: 
    Writing list of issue data to JSON...

Mon, 06/14/21, 09:19:57 PM: 
    Specified directory does not exist! Creating directory... 

Mon, 06/14/21, 09:19:57 PM:  Complete! 

Mon, 06/14/21, 09:19:57 PM:  Complete! 

Mon, 06/14/21, 09:19:57 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:23:38 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:23:38 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:23:38 PM:  Complete! 

Mon, 06/14/21, 09:23:38 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:23:39 PM:  Complete! 

Mon, 06/14/21, 09:23:39 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 10
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:23:40 PM: 
Attempting program start...

Mon, 06/14/21, 09:23:40 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 09:23:40 PM:  Complete! 

Mon, 06/14/21, 09:23:40 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 09:23:40 PM:  Complete! 

Mon, 06/14/21, 09:23:40 PM: 
    Getting issue data...

Mon, 06/14/21, 09:23:44 PM:  Complete! 

Mon, 06/14/21, 09:23:44 PM: 
    Writing list of issue data to JSON...

Mon, 06/14/21, 09:23:44 PM: 
    Specified directory does not exist! Creating directory... 

Mon, 06/14/21, 09:23:44 PM:  Complete! 

Mon, 06/14/21, 09:23:44 PM:  Complete! 

Mon, 06/14/21, 09:23:44 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:24:46 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:24:46 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:24:46 PM:  Complete! 

Mon, 06/14/21, 09:24:46 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:24:47 PM:  Complete! 

Mon, 06/14/21, 09:24:47 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 10
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:24:51 PM: 
Attempting program start...

Mon, 06/14/21, 09:24:51 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 09:24:51 PM:  Complete! 

Mon, 06/14/21, 09:24:51 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 09:24:51 PM:  Complete! 

Mon, 06/14/21, 09:24:51 PM: 
    Getting issue data...

Mon, 06/14/21, 09:24:55 PM: 
    Unspecified exception:

Traceback (most recent call last):
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 135, in main
    exe_menu( cfg_list, session, logger )
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 352, in exe_menu
    issue_metalist = get_issue_info( session, issue_paged_list,
  File "/home/j/git/GitHub-Repo-Extractor/TEST_extractor.py", line 761, in get_issue_info
    if comments_paged_list.totalCount == 0:
  File "/home/j/.local/lib/python3.9/site-packages/github/PaginatedList.py", line 149, in totalCount
    headers, data = self.__requester.requestJsonAndCheck(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 354, in requestJsonAndCheck
    *self.requestJson(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 454, in requestJson
    return self.__requestEncode(cnx, verb, url, parameters, headers, input, encode)
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 528, in __requestEncode
    status, responseHeaders, output = self.__requestRaw(
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 555, in __requestRaw
    response = cnx.getresponse()
  File "/home/j/.local/lib/python3.9/site-packages/github/Requester.py", line 127, in getresponse
    r = verb(
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 555, in get
    return self.request('GET', url, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 542, in request
    resp = self.send(prep, **send_kwargs)
  File "/usr/lib/python3.9/site-packages/requests/sessions.py", line 655, in send
    r = adapter.send(request, **kwargs)
  File "/usr/lib/python3.9/site-packages/requests/adapters.py", line 439, in send
    resp = conn.urlopen(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 699, in urlopen
    httplib_response = self._make_request(
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 445, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/usr/lib/python3.9/site-packages/urllib3/connectionpool.py", line 440, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.9/http/client.py", line 1345, in getresponse
    response.begin()
  File "/usr/lib/python3.9/http/client.py", line 307, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.9/http/client.py", line 268, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.9/socket.py", line 704, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.9/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.9/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
KeyboardInterrupt

Mon, 06/14/21, 09:24:55 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------


Mon, 06/14/21, 09:25:15 PM: 
----------------------------------------------------------------------------------------------------------------------
START OF PROGRAM RUN

Mon, 06/14/21, 09:25:15 PM: 
    Read configuration and initialize logging...

Mon, 06/14/21, 09:25:15 PM:  Complete! 

Mon, 06/14/21, 09:25:15 PM: 
    Validating user authentification...

Mon, 06/14/21, 09:25:16 PM:  Complete! 

Mon, 06/14/21, 09:25:16 PM:  
    PROGRAM START
    -------------
    Config used: 
        - config file name : configs/jabref/jabref_100.txt
        - repo             : JabRef/jabref
        - auth file        : auths/mcauley_auth.txt
        - rows             : 10
        - issue state      : closed        
        - pr state         : closed          
        - diagnostics      : true          
        - log file         : log_files/jabref/jabref_100_log.txt        
        - issue json file  : outputs/jabref_100/jabref_100_issue_list.json
        - pr JSON file     : outputs/jabref_100/jabref_100_pr_list.json
        - commit JSON file : outputs/jabref_100/jabref_100_commit_list.json
        - master JSON file : outputs/jabref_100/jabref_100_master_list.json
        - "pr" CSV file    : outputs/jabref_100/jabref_100_pr_output.csv       
        - "commit" CSV file: outputs/jabref_100/jabref_100_commit_output.csv


Mon, 06/14/21, 09:25:17 PM: 
Attempting program start...

Mon, 06/14/21, 09:25:17 PM: 
    Getting paginated list of issues...

Mon, 06/14/21, 09:25:17 PM:  Complete! 

Mon, 06/14/21, 09:25:17 PM: 
    Validating row quantity config for issue data collection...

Mon, 06/14/21, 09:25:17 PM:  Complete! 

Mon, 06/14/21, 09:25:17 PM: 
    Getting issue data...

Mon, 06/14/21, 09:25:21 PM:  Complete! 

Mon, 06/14/21, 09:25:21 PM: 
    Writing list of issue data to JSON...

Mon, 06/14/21, 09:25:21 PM:  Complete! 

Mon, 06/14/21, 09:25:21 PM: 
END OF PROGRAM RUN
----------------------------------------------------------------------------------------------------------------------

